File: sdkwork-backend-sdk-typescript/src/types/msg-video-content.ts
import type { VideoMediaResource } from './video-media-resource';

/** IMè§é¢æ¶æ¯åå®¹ç±» */
export interface MsgVideoContent {
  /** æ¶æ¯åå®¹ */
  content?: string;
  /** è§é¢èµæº */
  resource?: VideoMediaResource;
}

File: sdkwork-backend-sdk-java/src/main/java/com/sdkwork/backend/model/MsgVideoContent.java
package com.sdkwork.backend.model;

public class MsgVideoContent {
    private String content;
    private VideoMediaResource resource;

    public String getContent() {
        return this.content;
    }
    
    public void setContent(String content) {
        this.content = content;
    }

    public VideoMediaResource getResource() {
        return this.resource;
    }
    
    public void setResource(VideoMediaResource resource) {
        this.resource = resource;
    }
}
